.rubocop_todo.yml

```diff
@@ -8,7 +8,18 @@
 
 # Offense count: 9
 Metrics/AbcSize:
-  Max: 83
```

Comment:
This setting wasn't taken into account because it was overridden by a block in the `.rubocop.yml`. For now, I removed the block from `.rubocop.yml` and added violations to the `Exclude` section. The alternate solution can be setting `Max` to a higher value.